lib/ClangImporter/ClangAdapter.cpp

```diff
@@ -563,6 +563,9 @@ OptionalTypeKind importer::translateNullability(clang::NullabilityKind kind) {
   case clang::NullabilityKind::Unspecified:
     return OptionalTypeKind::OTK_ImplicitlyUnwrappedOptional;
   }
+
+  // Work around MSVC warning: not all control paths return a value
+  llvm_unreachable("All switch cases are covered");
```

Comment:
All of these make me think that we really need an `LLVM_MSVC_UNREACHABLE` macro that expands to `llvm_unreachable("silence MSVC warnings")` on MSVC and expands to nothing elsewhere.